README.md

```diff
@@ -465,6 +465,217 @@ node followerDiff.js
+| .[]
+| .[0].host as $host
+| "echo \(map(.rs) | join(" ")) > /tmp/rs-to-scan"
+| @sh "ssh -n \($user)@\($host) \(.)"'
+
+for RS in $RS_LIST; do
+    curl -s http://${BUCKETD_ENDPOINT}/_/raft_sessions/${RS}/info \
+    | jq --arg rs ${RS} "${JQ_SCRIPT_EACH}";
+done | jq -rs --arg user ${USER} "${JQ_SCRIPT_ALL}"
+```
+
+Executing this script gives on the standard output, the list of ssh
+commands to run to save the list of raft sessions to scan in a
+temporary file on each active stateful member, for example:
+
+```
+ssh -n 'root'@'storage-1' 'echo 3 4 5 7 > /tmp/rs-to-scan'
+ssh -n 'root'@'storage-2' 'echo 1 2 3 5 6 7 8 > /tmp/rs-to-scan'
+ssh -n 'root'@'storage-3' 'echo 1 2 3 4 5 6 8 > /tmp/rs-to-scan'
+ssh -n 'root'@'storage-4' 'echo 1 2 3 4 5 6 7 8 > /tmp/rs-to-scan'
+ssh -n 'root'@'storage-5' 'echo 1 2 4 6 7 8 > /tmp/rs-to-scan'
+```
+
+Check that what you get looks like the above, then execute those
+commands on the shell (e.g. by piping with `| bash -x`).
+
+**The following steps are to be run for each metadata stateful node.**
+
+### Step 4: SSH to next stateful host to scan
+
+```
+USER=root
+HOST=storage-1
+ssh ${USER}@${HOST}
+```
+
+### Step 5: Stop all follower repd processes
+
+In order to be able to scan the databases, follower repd processes
+must be stopped with the following command:
+
+```
+for RS in $(cat /tmp/rs-to-scan); do
+    docker exec -u root scality-metadata-bucket-repd \
+    supervisorctl -c /conf/supervisor/supervisord-repd.conf stop repd:repd_${RS}
+done
+```
+
+### Step 6:
+
+Run the "CompareRaftMembers/followerDiff" tool to generate a
+line-separated JSON output file showing all differences found between
+the leader and the local metadata databases, for raft sessions where
+the repd process is a follower.
+
+Example command:
+
+```
+DATABASES_GLOB=$(cat /tmp/rs-to-scan | tr -d '\n' | xargs -d' ' -IRS echo /databases/RS/0/*)
+
+mkdir -p followerDiff-results
+docker run --net=host --rm \
+-e 'BUCKETD_HOSTPORT=localhost:9000' \
+-v "/scality/ssd01/s3/scality-metadata-databases-bucket:/databases" \
+-e "DATABASES_GLOB=${DATABASES_GLOB}" \
+-v "${PWD}/followerDiff-digests:/digests" \
+-e "LISTING_DIGESTS_INPUT_DIR=/digests" \
+-v "${PWD}/followerDiff-results:/followerDiff-results" \
+-e "DIFF_OUTPUT_FILE=/followerDiff-results/followerDiff-results.jsonl" \
+registry.scality.com/s3utils/s3utils:1.13.0 \
+bash -c 'DATABASES=$(echo $DATABASES_GLOB) node CompareRaftMembers/followerDiff' \
+| tee -a followerDiff.log
+```
+
+### Step 7: Restart all stopped repd processes
+
+```
+for RS in $(cat /tmp/rs-to-scan); do
+    docker exec -u root scality-metadata-bucket-repd \
+    supervisorctl -c /conf/supervisor/supervisord-repd.conf start repd:repd_${RS}
+done
+```
+
+**If there are more active stateful hosts to scan, continue with Step 4
+on the next active stateful host.**
+
+### Step 8: Gather results
+
+Finally, once all scans have been done on all active stateful hosts,
+gather all diff results for later analysis.
+
```

Comment:
Could include a bit about what to expect in the results. Mine are empty files which I assume means that there are no differences. No need to get too technical just something high level about the different cases.

Reply:
I can add a pointer to where the output format is described above in the `followerDiff` tool description. Indeed empty files mean no difference (one difference per line)